nvflare/app_common/workflows/model_controller.py

```diff
@@ -94,6 +95,7 @@ def send_model(
         Returns:
             None
         """
+        self.set_fl_context(data)
```

Comment:
or should we add this in the "broadcast_model" of the "BaseModelController", what is the difference?

Reply:
That's right. We should do everything in as low level as possible. BaseModelController is a better place. And we probably need to call set_fl_context() in broadcast_model as well.

Reply:
Moved the self.set_fl_context(data) call to broadcast_model(). Also removed the no used logic to create the dummy FLModel object.